src/pages/projects/ProjectDetails.tsx

```diff
@@ -0,0 +1,106 @@
+import { Project, ProjectType, Skill } from "../../utils/types";
+import { Link } from "react-router-dom";
+import SkillButton from "../../components/SkillButton";
+import { IconArrowLeft, IconBrandGithub } from "@tabler/icons-react";
+import IconButton from "../../components/IconButton";
+import MonitorCanvas from "./MonitorCanvas";
+import PhoneCanvas from "./PhoneCanvas";
+import ProjectNavigator from "./ProjectNavigator";
+
+type ProjectDetailsProp = {
+  isDarkMode: boolean;
+  isMobile: boolean;
+  project: Project;
+};
+
+function ProjectDetails({ isMobile, isDarkMode, project }: ProjectDetailsProp) {
+  return (
+    <section
+      className={`grid ${
+        isMobile ? "grid-cols-1" : "grid-cols-[10%,70%,15%]"
+      } gap-7 p-5 justify-center mt-[100px]`}
+    >
+      <div className="flex flex-row">
+        <Link
+          to="/project/all"
+          className="flex space-x-2"
+          onClick={() => {
+            window.scrollTo({ top: 0 });
+          }}
+        >
+          <IconArrowLeft className="bg-royal-blue-100 rounded-lg text-white" />
+          <span>Back</span>
+        </Link>
+      </div>
+
+      <div className={`grid ${isMobile ? "" : "grid-cols-[60%,40%]"} gap-5`}>
```

Comment:
This logic can be made cleaner. Something like `${isMobile && "grid-cols-[60%,40%]"}`